package.json

```diff
@@ -22,7 +22,7 @@
     "./dist/browser.es2015.es.js": "./dist/browser.es2017.es.js"
   },
   "dependencies": {
-    "fusion-plugin-csrf-protection": "^0.3.1"
+    "fusion-plugin-csrf-protection": "^0.3.3"
```

Comment:
Will this release work given there's other changes?

Reply:
Should be fine I think